mousestyles/tests/test_kde.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from scipy.stats.distributions import norm
+from mousestyles.kde import kde
```

Comment:
I think it is much more consistent to have this as:
```python
from mousestyles import kde
```

And then below have an explicit reference using the `kde` function from the `kde` module:

```python
pdf = kde.kde(x=np.array([2, 3, 1, 0]), x_grid=np.linspace(0, 5, 10))
```